content/en/docs/faq/troubleshooting.md

```diff
@@ -128,3 +128,35 @@ For example:
 helm list -v 6
 ```
 
+### Tiller installations stopped working and access is denied
+
+Helm releases used to be available from [https://storage.googleapis.com/kubernetes-helm/](https://storage.googleapis.com/kubernetes-helm/). As explained in ["Announcing get.helm.sh"](https://helm.sh/blog/get-helm-sh/), the official location changed in June 2019. [GitHub Container Registry](https://github.com/orgs/helm/packages/container/package/tiller) makes all the old Tiller images available.
```

Comment:
nit: A more efficient form for

```
[https://storage.googleapis.com/kubernetes-helm/](https://storage.googleapis.com/kubernetes-helm/)
```

is

```
<https://storage.googleapis.com/kubernetes-helm>
```

which looks like this: <https://storage.googleapis.com/kubernetes-helm>

This short-form version automatically creates a URL link, referencing itself.